src/index.html

```diff
@@ -13,9 +13,41 @@
       "
     >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img class="card__img" 
+      src="images/imac.jpeg" 
+      alt="Apple A1419 iMac 27 inch Retina 5K Monoblock" 
+      height="134" 
+      width="160"
+      >
+
+      <p class="card__product-name">APPLE A1419 iMac 27" Retina <br> 5K Monoblock (MNED2UA/A)</p>
+      <p class="card__product-code">Product code: 195434</p>
+
+      <div class="rating card__rating">
```

Comment:
You don't need an additional rating block. Refactor it, please 
```suggestion
      <div class="card__rating">
```